### PR TITLE
chore: bump addon_docs @jest/transform

### PR DESCRIPTION
**Pull request by trfv**
Issue:

## What I did
bump addon_docs @jest/transform to fix this warning
```
npm WARN deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
```

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: addons/docs/package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
   "dependencies": {
     "@babel/plugin-transform-react-jsx": "^7.12.12",
     "@babel/preset-env": "^7.12.11",
-    "@jest/transform": "^26.6.2",
+    "@jest/transform": "^27.5.1",
```
Comment (shilman) [inline addons/docs/package.json:60]:
```suggestion
    "@jest/transform": "^26.6.2 || ^27.5.1",
```

**Review by shilman — CHANGES_REQUESTED** [review]
Thanks @trfv! This is technically a breaking change for Jest 26 users. If you update to accept either 26 or 27, it should achieve your intended results AND not be breaking. WDYT?

**Review by shilman — COMMENTED** [review]
@trfv thanks for making those changes. can you please try updating the snapshots to see if the build passes that way?

```
yarn bootstrap --core
yarn test -u
```

**Comment by trfv** [comment]
> @trfv thanks for making those changes. can you please try updating the snapshots to see if the build passes that way?
> 
> ```
> yarn bootstrap --core
> yarn test -u
> ```

hi @shilman 
thank you for your kind support.
`yarn bootstrap --core` did not make any change.
`yarn test -u` made some update for snapshots.

**Pull request closed without merging**